src/laboratory/api/serializers.py

```diff
@@ -15,6 +15,8 @@
 from django_filters import DateFromToRangeFilter, DateTimeFromToRangeFilter, filters
 from djgentelella.fields.drfdatetime import DateRangeTextWidget, DateTimeRangeTextWidget
 from django_filters import FilterSet
+import logging
```

Comment:
this import and the logger are not necessary here since they are not used, please keep the code clean, delete not used imports.